rpms/SPECS/3.11.0/wazuh-api-3.11.0.spec

```diff
@@ -0,0 +1,220 @@
+Summary:     Wazuh API is an open source RESTful API to interact with Wazuh from your own application or with a simple web browser or tools like cURL
+Name:        wazuh-api
+Version:     3.11.0
+Release:     %{_release}
+License:     GPL
+Group:       System Environment/Daemons
+Source0:     %{name}-%{version}.tar.gz
+URL:         https://www.wazuh.com/
+BuildRoot:   %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+Vendor:      Wazuh, Inc <info@wazuh.com>
+Packager:    Wazuh, Inc <info@wazuh.com>
+Requires(pre):    /usr/sbin/groupadd /usr/sbin/useradd
+Requires(post):   /sbin/chkconfig
+Requires(preun):  /sbin/chkconfig /sbin/service
+Requires(postun): /sbin/service
+
+Requires: nodejs >= 4.6
+Requires: wazuh-manager = 3.10.0
```

Comment:
The requires dependency is wrong.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/254/commits/bd9469d9104aaced5b40e1a09a527bbe1cefa707